app/lib/widgets/app_bar/app_bar.dart

```diff
@@ -61,7 +73,7 @@ class FlexAppBar extends StatelessWidget implements PreferredSizeWidget {
 }
 
 @widgetbook.UseCase(
-  name: 'Default',
+  name: '1-Default',
```

Comment:
For consistency with other PR widgetbook names

```suggestion
  name: '1. Default',
```